spec/models/manageiq/providers/openstack/storage_manager/cinder_manager/cloud_volume_spec.rb

```diff
@@ -86,7 +86,7 @@
       end
 
       it "validates the volume update operation when ems is missing" do
-        expect(cloud_volume).to receive(:ext_management_system).and_return(nil)
+        expect(cloud_volume).to receive(:ext_management_system).at_least(1).times.and_return(nil)
```

Comment:
Think it would be better to add a context and set up the cloud_volume to not have an ems_id?  Stubbing association calls seems more fragile
```
context "with a cloud_volume without an EMS" do
  let(:cloud_volume) { FactoryBot.create(:cloud_volume_openstack) }

  it "" 
    etc...
end
```

Reply:
This association stubbing was there before this PR, though, right?

Reply:
Yes

Reply:
I'll fix. thanks